discoveries/wq-models.discoveries.mdx

```diff
@@ -0,0 +1,219 @@
+    
+    Where the Mississippi River empties into the Gulf of Mexico, increased nutrients and sediment carried with the runoff contribute to hypoxia, a low-oxygen or “dead zone,” along the Texas-Louisiana shelf waters.
+
+    With increasing agricultural runoff from extreme rainfall events and other sources, the size of this dead zone has expanded in recent years. This is a global problem, with more than 400 such hypoxic zones identified worldwide.
+
+  </Prose>
+</Block>
+
+<Block>
+  <Prose>
+    ## Climate Change is Becoming Lethal to Aquatic Habitat
+
+	Water quality problems are also worsening with recurring droughts and wildfires. Research shows that high stream temperatures and post-fire sediment runoff threaten native fish habitat in the western United States.
+
+  </Prose>
+  <Figure>
+    <Image
+      src={new URL("./dead_chum_salmon.jpeg", import.meta.url).href}
+      alt="Picture of a dead Alaska Salmon due to increasing temperatures. "
+    />
+    <Caption
+      attrAuthor='Stephanie Quinn-Davidson'
+      attrUrl='https://news.climate.columbia.edu/2019/09/06/warming-rivers-salmon-die-offs/'
+    >
+      Dead Alaskan chum salmon in the Koyukuk River in July 2019.
+    </Caption>
+  </Figure>
+</Block>
+
+
+<Block>
+  <Figure>
+    <Image
+      src={new URL("./USGS_WQSites.png", import.meta.url).href}
+      alt="Real Time USGS Nitrate Monitoring Stations"
+    />
+    <Caption
+      attrAuthor='USGS'
+      attrUrl='https://waterwatch.usgs.gov/wqwatch/?pcode=00630'
+    >
+      Continuous Real Time Surface Water Quality Measurement Sites in the United States
+    </Caption>
+  </Figure>
+  <Prose>
+    ## Barrier to Solving Water Quality Problems ‒ Limited or No Data
+
+	Across the more than two million rivers and streams in the Continental U.S., there are only a few hundred continuous water quality monitoring sites. This data gap limits our ability to understand where, when, and how worsening water quality will adversely impact human uses and aquatic health.
+
+  </Prose>
+</Block>
+
+
+<Block>
+  <Prose>
+    ## A New NASA Model Brings Open Science to Target Water Quality Problems
+
+    This EIS project is the first effort to integrate NASA’s Land Information System with a globally used process-based water quality simulation model - Soil & Water Assessment Tool.
+
+    Comparing model results with ground observations shows the ability of this framework to predict water quality at high resolution river network-scales across the Mississippi River Basin. With a generalizable open science design, this framework can be reproduced for other major basins of the world. 
+
+  </Prose>
+</Block>
+
+
+<Block>
+  <Prose>
+    ## The First Spatially Distributed & Temporally Continuous Water Quality Estimates for US Rivers
+  </Prose>
+  <Figure>
+    <Image
+      src={new URL("./Animated_NO3_2018.discoveries.gif", import.meta.url).href}
+      alt="Nitrate Loads across the United States in 2018"
+    />
+    <Caption
```

Comment:
👋 I am not sure if you are actively working on this or not, but some Caption tags (including this one) were not closed properly. (This should be `<Caption> `

Reply:
Hello, I am actively working on this, However, I am still going through a steep learning curve as I am not extensively familiar with this. I tried your suggestion on closing the caption tags and the story I wanted to try was successful. I am still working on improving my code so you might see more trial pull request (so that i can check the preview). Thank you for your suggestions.